gtsam_unstable/examples/FixedLagSmootherExample.cpp

```diff
@@ -145,5 +145,18 @@ int main(int argc, char** argv) {
     cout << setprecision(5) << "    Key: " << key_timestamp.first << "  Time: " << key_timestamp.second << endl;
   }
 
+  // get the linearization point
+  Values result = smootherISAM2.calculateEstimate();
+
+  // create the factor graph
+  auto &factor_graph = smootherISAM2.getFactors();
```

Comment:
Nit: in c++ we now try to follow the Google conventions, so factor graph and linear graph should become camelCase.